docker/release/opensuse13.2/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Docker image file that describes an OpenSUSE 13.2 (AKA harlequin) image with PowerShell installed from PowerShell Release
+FROM opensuse:harlequin
+LABEL maintaner Travis Plunk <travis.plunk@microsoft.com>
```

Comment:
Do we need the Maintainer tag? I thought LABEL replaced MAINTAINER https://github.com/docker/docker/pull/25466